### ISSUE TITLE
Testen

### ISSUE THREAD

**Issue by Rox-Codelabs** [Rox-Codelabs/the-startup-responsive-interactive-website#4]
WCAG Audit

Color contrast test

HTML Validator

**Comment by roxyfokker** [comment]
De Lighthouse-test geeft een score van 100 voor accessibility
<img width="1873" height="850" alt="Image" src="https://github.com/user-attachments/assets/16ef77c0-b2fc-4a66-97b3-326e9013f161" />
Lighthouse-score is 100, maar linkbeschrijvingen kunnen duidelijker en consistenter volgens WCAG.

**Comment by roxyfokker** [comment]
PASSED:
<img width="250" height="800" alt="Image" src="https://github.com/user-attachments/assets/b394518b-6bd4-486a-a526-4e98e9dd1fe2" /> <img width="250" height="800" alt="Image" src="https://github.com/user-attachments/assets/f742c87e-4cd9-41fa-89be-853a3bb7d6c6" />

FAILED:
<img width="250" height="800" alt="Image" src="https://github.com/user-attachments/assets/e68c3924-d00d-47fe-a090-477437025eee" /> <img width="250" height="800" alt="Image" src="https://github.com/user-attachments/assets/c341755c-0ba6-4401-bfcd-7ba1a9f69bbd" />


Alternative:

<img width="250" height="800" alt="Image" src="https://github.com/user-attachments/assets/fbd18eec-0b40-4110-93a2-9987f346c4c8" />

Uit de color contrast test bleek dat de website nog niet volledig voldoet aan de juiste toegankelijkheidsstandaarden. Omdat de opdrachtgever echter specifiek aangaf het design niet te veranderen, konden we dit niet aanpassen. Wel hebben we het tijdens de print review besproken en advies gegeven over mogelijke verbeteringen.

**Comment by roxyfokker** [comment]
Tijdens de handmatige WCAG-test kwam een groot probleem naar voren: wanneer je met de tab-toets door de website navigeert, wordt het menu toch bereikt terwijl het niet zichtbaar is. Dit moet ik oplossen door het menu display: none te geven wanneer het verborgen is of door de tabindex van de elementen aan te passen.

**Comment by roxyfokker** [comment]
HTML validator geeft geen problemen meer aan
<img width="1804" height="30" alt="Image" src="https://github.com/user-attachments/assets/7c4619eb-ae5f-4f09-b385-0180d059285d" />
